packages/commcare/src/Adaptor.js

```diff
@@ -347,6 +348,119 @@ export function request(method, path, body, options = {}) {
     }
   };
 }
+
+/**
+ * Bulk upload data to CommCare. Accepts an array of objects, converts them into
+ * an XLS representation, and uploads.
+ * @public
+ * @function
+ * @example <caption>Upload a single row of data for case-data</caption>
+ * bulk(
+ *    [
+ *      {name: 'Mamadou', phone: '000000'},
+ *    ],
+ *    {
+ *      case_type: 'student',
+ *      search_field: 'external_id',
+ *      create_new_cases: 'on',
+ *    }
+ * )
+ * @example <caption>Upload a single row of data for a look-up table</caption>
+ * bulk(
+ *     'lookup-table'
+ *  {
+ *    types: [{
+ *
+ *   'DELETE(Y/N)':'N',
+ *   table_id: 'obat',
+ *   'is_global?':'yes',
+ *   'field 1': 'Nama',
+ *   'field 2': 'Satuan',
+ *   'field 3': 'Harga',
+ *   'field 4': 'kfa_codes',
+ *   'field 5': 'satusehat_id',
+ *   'field 6': 'strength',
+ *   'field 7': 'strength_unit',
+ *      }],
+ *      obat: [{
+ *       UID: 'bb73d4706adf47308c0cb16b9df74d03',
+ *        'DELETE(Y/N)':'N',
+ *       'field:Nama': 'ACARBOSE PRB BPJS 100 mg*',
+ *        'field:Satuan': 'TABLET',
+ *        'field:Harga':'1375',
+ *        'field:kfa_codes': '92000372',
+ *        'field:satusehat_id': 'TAB',
+ *        'field:strength': '100',
+ *        'field:strength_unit': 'mg',
+ *     }],
+ *   }
+ * )
+ * @param {string} type - case-data or lookup-tables
```

Comment:
`lookup-tables` or `lookup-table` ?

Reply:
@josephjclark  I have fixed it. Its `lookup-table`